src/swap/types.ts

```diff
@@ -219,6 +228,7 @@ export type SwapUnit = {
   balance?: string;
   error?: SwapError;
   loading: boolean;
+  response: UseBalanceReturnType | UseReadContractReturnType | undefined;
```

Comment:
nit. we can just add `response?` to the type, and that will be as `undefined`. So it will be
```
response?: UseBalanceReturnType | UseReadContractReturnType;
```

Reply:
done